flink/v1.15/flink/src/main/java/com/netease/arctic/flink/read/hybrid/assigner/StaticSplitAssigner.java

```diff
@@ -0,0 +1,133 @@
+package com.netease.arctic.flink.read.hybrid.assigner;
```

Comment:
Lack of Apache License at the head of the class.

You cloud add the Apache license at the head of each new file.

The following steps may help you automate in IntelliJ by adding a Copyright profile:

1、Go to “Settings” → “Editor” → “Copyright” → “Copyright Profiles”.

2、Add a new profile and name it “Apache”.

3、Add the following text as the license text:

4、Go to “Editor” → “Copyright” and choose the “Apache” profile as the default profile for this project.

5、Click “Apply”.